pymc/sampling_jax.py

```diff
@@ -94,14 +95,17 @@ def get_jaxified_graph(
     return jax_funcify(fgraph)
 
 
-def get_jaxified_logp(model: Model) -> Callable:
+def get_jaxified_logp(model: Model, sampler=None) -> Callable:
 
     logp_fn = get_jaxified_graph(inputs=model.value_vars, outputs=[model.logpt()])
```

Comment:
If we do it like this we don't need the if/else in `logp_fn_wrap`:

```suggestion
    model_logpt = model.logpt()
    if negative_logp:
        model_logpt = -model_logpt
    logp_fn = get_jaxified_graph(inputs=model.value_vars, outputs=[model_logpt])
```

Reply:
That looks nice!